src/ring_theory/ideal_operations.lean

```diff
@@ -655,11 +712,21 @@ variables {R : Type u} [comm_ring R]
 def jacobson (I : ideal R) : ideal R :=
 Inf {J : ideal R | I ≤ J ∧ is_maximal J}
 
```

Comment:
Maybe it would make sense to move this to its own file? `ring_theory/jacobson_ideal.lean`?

We've had some big refactors recently to split overly large files, and it's a lot easier to do it before it becomes a problem!

Reply:
I went ahead and did this, and moved a couple more definitions as well to prevent circular dependencies between the new and old files.